modules/05 ES6 and Typescript/Assignments/4/assignment22.ts

```diff
@@ -0,0 +1,33 @@
+interface IStringUtils{
+    getVowels(str:string):number;
+    removeChar(str:string,pos:number):string;
+    incrementLetters(str:string):string;
+}
+
+class StringUtils implements IStringUtils{
+    constructor(){}
+    public getVowels(str:string){
+        let count:number = 0;
+        let vowels:string = "aeiou";
+        let res:string = str.toLowerCase();
+        for(let i:number=0; i< res.length; i++){
+            if(vowels.includes(res[i]))
+                count++;
+        }
+        return count;
+    };
+    public removeChar(str:string,pos:number){
+        let newStr:string = str.replace(str[pos],'');
+        return newStr;
+    };
+    public incrementLetters(str:string){
+        let newStr:string = "";
+        for(let i:number=0; i<str.length; i++){
+            if(str[i] === " ")
+                newStr += " ";
+            else
+                newStr += String.fromCharCode(str.charCodeAt(i) + 1);
+        }
+        return newStr;
+    };
+}
```

Comment:
correct. please do not submit unrelated code (e.g. modules/03 Front End Development/Assignments/07/answer.html)